pages/about/data_policy.vue

```diff
@@ -43,7 +43,7 @@
           </template>
         </i18n>
         <v-card>
-          <v-card-title class="accent">
+          <v-card-title class="info">
```

Comment:
As with the colour theme, let's simply remove these classes.

When I was looking at the purpose of "cards" in material design, they're suppose to be uniform in colour background (except for optional images): https://material.io/components/cards

Since these cards are providing important "informational" notes/context in the page, I think using the `color="info"` attribute in the card is appropriate:
```
<v-card color="info">
```

Likewise for the other cards.